stdlib/public/stubs/Stubs.cpp

```diff
@@ -140,12 +140,21 @@ static int swift_snprintf_l(char *Str, size_t StrSize, locale_t Locale,
 #endif
 
 template <typename T>
+static int swift_floatingPointToStringPrecision(bool Debug) {
```

Comment:
It looks to me like a good alternative to code like this:
```cpp
  int Precision = std::numeric_limits<T>::digits10;
  if (Debug) {
    Precision = std::numeric_limits<T>::max_digits10;
  }
```
which adds complexity for the reader - it adds a 5th conditional to the function.